src/product_info/product_info.jsx

```diff
@@ -146,7 +165,7 @@ const ProductInfo = () => {
                         <button className='amount_calc_button' onClick={raise}>+</button>
                     </div>
                     <div className='add_compare'>
-                        <button className='add_cart add_compare_btn'>Add To Cart</button>
+                        <button className='add_cart add_compare_btn' onClick={() => addProductHandler()}>Add To Cart</button>
```

Comment:
so why do you add to cart a hardcoded product (lines 70-79) instead of adding the product on which this button is clicked?